pkg/cmd/cli/install/install.go

```diff
@@ -291,5 +295,9 @@ func (o *InstallOptions) Validate(c *cobra.Command, args []string, f client.Fact
 		return errors.New("Cannot use both --secret-file and --no-secret")
 	}
 
+	if o.DefaultResticMaintenanceFrequency < 0 {
```

Comment:
Should this not be `<= 0`?

Reply:
For `velero install` I didn't give this flag a default value, so the default is 0 (meaning don't explicitly specify this flag when creating the Deployment). So 0 is valid here.